packages/react-reconciler/src/SchedulerWithReactIntegration.new.js

```diff
@@ -147,10 +148,14 @@ export function scheduleSyncCallback(callback: SchedulerCallback) {
     // Flush the queue in the next tick, at the earliest.
     // TODO: Figure out how to remove this It's only here as a last resort if we
     // forget to explicitly flush.
-    immediateQueueCallbackNode = Scheduler_scheduleCallback(
-      Scheduler_ImmediatePriority,
-      flushSyncCallbackQueueImpl,
-    );
+    if (supportsMicrotasks) {
```

Comment:
@acdlite I know we wanted a killswitch for this, but this property is wired up to the feature flag we're currently experimenting. We also can't get completely rid of the below schedule immediate callback because not all host configs have a queueMicrotask impl.

Reply:
The ones that don't have a `queueMicrotask` can continue using `scheduleCallback`

Reply:
In which case, the extra killswitch should be fine, right?